src/librustdoc/core.rs

```diff
@@ -359,6 +364,9 @@ pub fn run_core(options: RustdocOptions) -> (clean::Crate, RenderInfo, RenderOpt
         ..Options::default()
     };
 
+    lazy_static! {
+        static ref EMPTY_MAP: FxHashSet<LocalDefId> = FxHashSet::default();
```

Comment:
`EMPTY_SET` would be more accurate. Also I suspect you can move this into the closure that returns it.